theia-ide/theia-apps
# yangster: remove 'yangster' image

#### What it does


Fixes: #388 

The following pull-request includes updates to **yangster**.

The `yangster-docker` image is removed since it makes use of the outdated `yangster` Theia-extension (which was replaced by the [`yang-vscode`](https://github.com/theia-ide/yang-vscode) vscode extension. The `theia-full` image is updated to make use of the extension instead (so there is no need for the individual image anymore).

#### How to test


The `yangster-docker` image is removed so verify that there are no longer any references to `yangster` besides the new vscode extension. The `full-image` should now support the new extension.

The build currently fails due to #386 

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
